healthcare/fhir_store_patch.go

```diff
@@ -36,11 +36,11 @@ func patchFHIRStore(w io.Writer, projectID, location, datasetID, fhirStoreID, to
 
 	name := fmt.Sprintf("projects/%s/locations/%s/datasets/%s/fhirStores/%s", projectID, location, datasetID, fhirStoreID)
 
-	if _, err := storesService.Patch(name, &healthcare.FhirStore{
-		NotificationConfig: &healthcare.NotificationConfig{
-			PubsubTopic: topicName, // format is "projects/*/locations/*/topics/*"
-		},
-	}).UpdateMask("notificationConfig").Do(); err != nil {
+	notificationConfig := &healthcare.FhirNotificationConfig{
+		PubsubTopic: topicName, // format is "projects/*/locations/*/topics/*"
```

Comment:
nit: I would prefer to have this comment above the field. Up to you!

Reply:
Done.